### PR TITLE
docs(appium): Added correct doc url path for images comparison and settings

### PR DESCRIPTION
**Pull request by bipin-k**
## Proposed changes

* Added correct doc path for images comparison and settings

## Types of changes

What types of changes does your code introduce to Appium?
- [x] Documentation Update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the CLA
- [x] Lint and unit tests pass locally with my changes

## Further comments
N/A

### CODE REVIEW

File: packages/appium/lib/utils.js [hunk 1]
```diff
@@ -280,7 +280,7 @@ function adjustNodePath() {
  * The capabilities argument itself gets mutated, so it does not contain parsed
  * settings anymore to avoid further parsing issues.
  * Check
- * https://github.com/appium/appium/blob/master/docs/en/advanced-concepts/settings.md
+ * https://appium.io/docs/en/2.0/guides/settings/
```
Comment (eglitise) [inline packages/appium/lib/utils.js:283]: For future compatibility it may be better to change this to `https://appium.io/docs/en/latest/guides/settings/`

Comment (bipin-k) [reply packages/appium/lib/utils.js:283]: @eglitise The url as mentioned, `https://appium.io/docs/en/latest/guides/settings/` is getting being redirected to `https://appium.io/docs/en/2.0/guides/settings/`. Do you still want me to make the changes as suggested by you?

Comment (eglitise) [reply packages/appium/lib/utils.js:283]: Yes, it's supposed to redirect, I'm just thinking of the possibility when the docs eventually get updated to 2.1, 2.2, etc, in which case this link may become outdated. @jlipps what do you think?

Comment (jlipps) [reply packages/appium/lib/utils.js:283]: yes, ideally all our doc links should refer to `latest` so that the redirect gets to the most current version.

**Review by jlipps — APPROVED** [review]

**Pull request merged**